manifests/config/global.pp

```diff
@@ -0,0 +1,17 @@
+# This private class manages Jenkins' config.xml
+#
+class jenkins::config::global {
```

Comment:
@rtyler Is 'global'  typical "jenkins speak" for the main `config.xml`?  The inconsistency between the API endpoint names, views, description, and html UI tab names drives me crazy.